qa/tasks/cephfs/mount.py

```diff
@@ -536,16 +536,9 @@ def run_as_user(self, args, user, wait=True, stdin=None,
 
     def run_as_root(self, args, wait=True, stdin=None, check_status=True,
```

Comment:
````
def run_as_root(self, **kwargs):
   return self.run_as_user(**kwargs)
````

? (Although you might need to add the stdout/stderr below.)

Reply:
That would be certainly be better. `stdout` and `stderr` can be added by `run_as_user()`, we'll just need to do `kwargs['user'] = 'root'`.